jp/components/icon.md

```diff
@@ -113,7 +113,7 @@ public ngOnInit() {
 </div>
 
 ### 6.2.0 の重大な変更
-* [`IgxIconComponent`]（{environment：angularApiUrl} /classes/igxiconcomponent.html） `iconName` プロパティは非推奨です。「マテリアル」アイコンのアイコン名を設定するには、開始タグと終了タグの間にアイコンの名前を配置します。'Font Awesome' と SVG のアイコンには `name` プロパティを使用します。
+* [`IgxIconComponent`]（{environment:angularApiUrl}/classes/igxiconcomponent.html） `iconName` プロパティは非推奨です。「マテリアル」アイコンのアイコン名を設定するには、開始タグと終了タグの間にアイコンの名前を配置します。'Font Awesome' と SVG のアイコンには `name` プロパティを使用します。
```

Comment:
@yoshimis the brackets of the link here are full width even though it's not visible in github's editor.